tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -29,6 +29,27 @@ HDMI1_ID=6,HDMI2_ID=7,HDMI3_ID=8"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-mtl-rt712-l0\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
+HDMI1_ID=3,HDMI2_ID=4,HDMI3_ID=5"
+
+"cavs-sdw\;sof-mtl-rt712-l0-4ch\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
+NUM_DMICS=4,PDM1_MIC_A_ENABLE=0,PDM1_MIC_B_ENABLE=0,DMIC0_ID=3,DMIC1_ID=4,\
```

Comment:
What is DMIC1_ID @bardliao ? If indeed the 2-ch version creates dmic01 and dmic16k, should we not have an ID for the BE ID for the dmic16k dailink?

Reply:
> What is DMIC1_ID @bardliao ? If indeed the 2-ch version creates dmic01 and dmic16k, should we not have an ID for the BE ID for the dmic16k dailink?

Hmm, DMIC1_ID is for the second DMIC DAI id. I thought the second DMIC DAI will not be created if NUM_DMICS=2. But apparently, it is always created. So, we need to set DMIC1_ID=4 in the -2ch topology, too.

Reply:
Thanks @bardliao, updated both parts.